### DISCUSSION TITLE
How can I help you?

### DISCUSSION THREAD

**Discussion by zmetedinler** [diegodlh/zotero-cita#65] [General]
I think I can translate your plugin to Turkish. Thus, Turkish users can easily use the plugin. If you help me with this, I can already translate the user manual into Turkish, starting from the menus. I will can also understand your problems while contributing to your project in this way.

**Comment by diegodlh** [comment]
Hi, @zmetedinler! Thank you very much for your interest! You can manually add a Turkish folder [here](https://github.com/diegodlh/zotero-wikicite/tree/master/static/chrome/locale), but I have just [requested](https://phabricator.wikimedia.org/T279972) to have a crowdsourced translation project set up at translatewiki.net. So I would suggest that we wait for that, as it'd be easier.

Regarding other help, I would really appreciate it if you could download, install and test the addon (just make sure you backup your Zotero database before, just in case) and report any issues you may find. That would be really helpful right now!

Finally, later in May there will be an English workshop introducing the tool and explaining how to use it. When that's available, you can also help adding Turkish subtitles. That would be amazing as well!

Thanks again!

**Comment by metedinler** [comment]
Mr. Diego
I'm ready for translation. Should I subscribe to Taraslatewiki and search for the cita project?

**Reply by diegodlh** [reply]
Hi, @metedinler! Exactly, you have to sign up to [translatewiki](https://translatewiki.net/) as a translator and then you can start contributing your translations to the [Cita](https://translatewiki.net/wiki/Translating:Cita) project. There are still 42% of messages non-translated to Turkish yet! Thank you!

**Reply by diegodlh** [reply]
By the way, @metedinler, maybe you are also interested in translating the Cita page, [here](https://www.wikidata.org/wiki/Wikidata:Zotero/Cita#Documentation). If so, just click on "Translate this page" above and choose Turkish to get started :) Thanks!!
